daytrader3-ee6/build.gradle

```diff
@@ -95,34 +88,29 @@ String getHttpPort() {
     return DEFAULT_HTTP_PORT
 }
 
-libertyStart.mustRunAfter publishEar
+installApps.dependsOn 'ear'
+libertyStart.mustRunAfter 'installApps'
 
 liberty {
     installDir = owner.hasProperty('libertyRoot') ? "${libertyRoot}" : null
-    serverName = 'daytrader3_Sample'
-    userDir = '../daytrader3-ee6-wlpcfg'
-    clean = true
-    install {
-       
+    server{
+        name = 'daytrader3_Sample'
+        userDir = rootProject.getProjectDir().getAbsolutePath()+'/daytrader3-ee6-wlpcfg'
+        looseApplication = false
+        apps = [file(ear.archivePath)]
+        features {
+           name = ['daytrader3_Sample']
+           acceptLicense = true
+        }
     }
 }
 
-liberty {
-	features {
-       name = ['daytrader3_Sample']
-      
-       acceptLicense = true
-    }
-} 
-
-installFeature.dependsOn(installLiberty)
-
-assemble.dependsOn('publishEar','copyDerbyLib')
-
-// If libertyRoot is not set, automatically download & install Liberty runtime
 if (!hasProperty('libertyRoot')) {
     fvt.dependsOn('installFeature')
 }
 
+assemble.dependsOn 'installApps','copyDerbyLib'
+
 // Run FVT by default
-check.dependsOn fvt
+check.dependsOn 'fvt'
+libertyCreate.enabled = false

```

Comment:
Add a comment to this line.

// do not create server since the usr directory is prebuilt.